docs/tabular_buckets_create_table.msc

```diff
@@ -0,0 +1,128 @@
+msc {
+    width=1800;
+    C [label="client"],
+    KVA [label="riak_kv_ts_api"],
+    CBT [label="riak_core_bucket_type"],
+    CC [label="riak_core_claimant"],
+    CM [label="riak_core_metadata"],
+    CMM [label="riak_core_metadata_manager"],
+    KVML [label="riak_kv_metadata_store_listener"],
+    KVNT [label="riak_kv_ts_newtype"];
+
+    C -> KVA [label="create table"];
+    KVA -> CBT [label="create"];
+    CBT -> CC [label="create_bucket_type"];
+    CC -> CC [label="claimant"];
+    CC -> CC [label="create_bucket_type"];
+    CC -> CC [label="get_bucket_type"];
+    CC -> CM [label="get"];
+    --- [label="bucket DNE"];
+    CC <- CM [label="<- get, undefined"];
```

Comment:
I'm confused about lines 20 and 21. 21 has two different arrow directions (perhaps it's showing both the call and the return?), and it's not clear why CC calls both CM:get and CM:get_bucket_type.